client/post-editor/editor-title/style.scss

```diff
@@ -36,14 +32,17 @@
 	}
 }
 
-.editor-title__input[type="text"] {
+.editor-title__input {
+	min-height: 0;
 	padding-left: 10px; /* Intentionally non-standard to align editor body margin (10px) */
 	padding-right: 10px;
 	border: none;
 	font-family: $serif;
 	font-size: 28px;
 	color: $gray-dark;
 	font-weight: 600;
+	resize: none;
+	-ms-overflow-y: hidden !important;
```

Comment:
>An ugly Windows native scrollbar, IIRC.

Indeed, this one: https://github.com/Automattic/wp-calypso/pull/4826#issuecomment-212604535